src/MagnumPlugins/DrFlacAudioImporter/DrFlacImporter.cpp

```diff
@@ -35,6 +35,46 @@
 
 namespace Magnum { namespace Audio {
 
+#define bfv(value) Buffer::Format::value
+namespace {
+
+    // number of channels = 8
+    // number of bits = 5
+    const Buffer::Format  flacFormatTable[8][5] = {
+        { bfv(Mono8),   bfv(Mono16),   bfv(MonoFloat),   bfv(MonoDouble)   },       // Mono
+        { bfv(Stereo8), bfv(Stereo16), bfv(StereoFloat), bfv(StereoDouble) },       // Stereo
+        { Buffer::Format{}, Buffer::Format{}, Buffer::Format{}, Buffer::Format{} }, // Not a thing
+        { bfv(Quad8), bfv(Quad16), bfv(Quad32), bfv(Quad32) },                      // Quad
+        { Buffer::Format{}, Buffer::Format{}, Buffer::Format{}, Buffer::Format{} }, // Also not a thing
+        { bfv(Surround51Channel8), bfv(Surround51Channel16), bfv(Surround51Channel32), bfv(Surround51Channel32) }, // 5.1
+        { bfv(Surround61Channel8), bfv(Surround61Channel16), bfv(Surround61Channel32), bfv(Surround61Channel32) }, // 6.1
+        { bfv(Surround71Channel8), bfv(Surround71Channel16), bfv(Surround71Channel32), bfv(Surround71Channel32) }  // 7.1
```

Comment:
Can you also update the documentation in the header file to mention all newly supported formats?